script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt

```diff
@@ -1209,8 +1209,8 @@ The three Principles buried in Ciel's body activate.
 [sha:292581bf223d2cb2810666caaaa585482067c69d]{
 -- Page 215, Offset 38960.
 -- 秘蔵していたヴローヴの原理血戒を新たな炉心として、シエルは魔力を補充する。
-After keeping it in reserve all this time,
- she uses Vlov's Hemonomic Principle as a fresh reactor core to supplement her magical energy.
+After having kept it in reserve all this time,
+ Ciel uses Vlov's Hemonomic Principle as a fresh reactor core to supplement her magical energy.
 }
```

Comment:
A pronoun would work here I think. Ciel's already mentioned in the page.
>The three Principles buried in Ciel's body activate.
After having kept it in reserve all this time,
 she uses Vlov's Hemonomic Principle as a fresh reactor core to supplement her magical energy.